exercises/wordy/example.rs

```diff
@@ -34,6 +42,11 @@ pub fn answer(c: &str) -> Option<i32> {
         .map(|word| {
             if let Ok(i) = word.parse::<i32>() {
                 Token::Number(i)
+            } else if let Some(i) = word
+                .get(..word.len() - 2)
+                .and_then(|word| word.parse::<i32>().ok())
+            {
+                Token::Number(i)
```

Comment:
if it were necessary to distinguish between these and the `Number`s above, I would have suggested `Token::Ordinal(i)`. However, since it is not necessary, this suggestion should instead be ignored.